datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxDatabaseRestoreStatusResponse.java

```diff
@@ -0,0 +1,25 @@
+package com.sequenceiq.sdx.api.model;
+
+public class SdxDatabaseRestoreStatusResponse {
+    private String status;
```

Comment:
what is the value set? If it is fixed then I would prefer using an enum, if not then it is fine.

Reply:
done